frontend/src/App.js

```diff
@@ -5,13 +5,10 @@
 import React, { useEffect, useState } from "react";
 import appwriteApi from "./api/appwriteApi";
 import { Route, BrowserRouter as Router } from "react-router-dom";
-
```

Comment:
Why removing these empty lines? I added them because I found them helpful to group related dependencies together.